src/features/ITGovernance/Admin/GRBReview/GRBReviewForm/Presentation/index.tsx

```diff
@@ -1,60 +1,509 @@
+      : undefined;
+
+    const presentationDeckFileData = links.presentationDeckFileData?.size
+      ? await fileToBase64File(links.presentationDeckFileData)
+      : undefined;
+
+    return mutateAsync({
+      variables: {
+        asyncRecordingDate: {
+          systemIntakeID: asyncRecordingDate.systemIntakeID,
+          grbReviewAsyncRecordingTime:
+            asyncRecordingDate.grbReviewAsyncRecordingTime || null
+        },
+        links: {
+          systemIntakeID: asyncRecordingDate.systemIntakeID,
+          transcriptFileData:
+            links.transcriptFileData === null ? null : transcriptFileData,
+          presentationDeckFileData:
+            links.presentationDeckFileData === null
+              ? null
+              : presentationDeckFileData,
+          recordingLink: links.recordingLink,
+          recordingPasscode: links.recordingPasscode,
+          transcriptLink: links.transcriptLink
+        }
+      }
+    });
+  };
 
   return (
-    <EasiFormProvider<PresentationFields> {...form}>
-      <GRBReviewFormStepWrapper<PresentationFields>
-        grbReview={grbReview}
-        // TODO: Update onSubmit to return mutation
-        onSubmit={(async () => null) as unknown as GRBReviewFormStepSubmit}
-      >
-        <Controller
-          name="grbDate"
-          control={control}
-          render={({ field: { ref, ...field }, fieldState: { error } }) => (
-            <FormGroup error={!!error}>
-              <Label
-                htmlFor={field.name}
-                className="text-normal"
-                requiredMarker
-              >
-                GRB Date
-              </Label>
-
-              <DatePicker
-                {...field}
-                id={field.name}
-                defaultValue={field.value}
-              />
-            </FormGroup>
-          )}
-        />
-      </GRBReviewFormStepWrapper>
-    </EasiFormProvider>
+    <>
+      {grbReview.grbReviewType === SystemIntakeGRBReviewType.STANDARD ? (
+        <EasiFormProvider<StandardPresentationFields> {...standardForm}>
+          <GRBReviewFormStepWrapper
+            grbReview={grbReview}
+            onSubmit={onSubmitStandard}
+          >
+            <Grid desktop={{ col: 6 }}>
+              <div className="border-top-1px border-base-lighter margin-top-3 easi-text-normal">
+                <h4 className="margin-top-1 margin-bottom-05">
+                  {t('presentationGRBReviewForm.heading')}
+                </h4>
+
+                <p className="margin-y-0 text-base">
+                  {t('presentationGRBReviewForm.description')}
+                </p>
+
+                <Alert type="info" slim className="margin-top-105">
+                  {t('presentationGRBReviewForm.alert')}
+                </Alert>
+
+                <Controller
+                  name="grbMeetingDate.grbDate"
+                  control={control}
+                  rules={{
+                    required: 'presentationGRBReviewForm.required'
```

Comment:
This error message needs to be translated.

```suggestion
                    required: t<string>('presentationGRBReviewForm.required')
```

Reply:
@patrickseguraoddball did you fix the error message translation? I'm not seeing it in the latest changes.